ui/src/pageLayout/containers/Nav.tsx

```diff
@@ -132,31 +131,29 @@ class SideNav extends PureComponent<Props, State> {
           )}
           active={getNavItemActivation(['tasks'], location.pathname)}
         />
-        <FeatureFlag name="alerting">
-          <NavMenu.Item
+        <NavMenu.Item
```

Comment:
Wait don't we still need the `<FeatureFlag>`?

Reply:
We do :)